packages/chargebee/README.md

```diff
@@ -78,6 +78,12 @@ const App = () => (
     {/* Should return a promise, that will resolve a hosted page object */}
     onHostedPageGet={() => Promise.resolve({ id: '', url: '' })}
 
+    {/* Will be called once the checkout page is loaded */}
+    onLoaded={() => {}}
+
+    {/* Will be called everytime an user navigates from one step to another */}
```

Comment:
Add chargbee doc page about the format of `currentStep` (if there is one)

Reply:
Added a reference to https://www.chargebee.com/docs/2.0/gtm.html -> cb-checkout

Ready to publish I would say.